Source/ASDisplayNode.mm

```diff
@@ -245,11 +246,6 @@ + (void)load
   ASScreenScale();
 }
 
-+ (BOOL)layerBackedNodesEnabled
-{
-  return YES;
```

Comment:
This method was never made public or used at all. It's been replaced by the instance method `-supportsLayerBacking` now.